<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportGraphicsDialog</class>
 <widget class="QDialog" name="ExportGraphicsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to Graphics File</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="_saveLocationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Save location:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;File format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>_comboBoxFormats</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="Kst::FileRequester" name="_saveLocation" native="true"/>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="Kst::ComboBox" name="_comboBoxFormats">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="title">
      <string>Save Options</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="_saveOnce">
        <property name="whatsThis">
         <string>Do not autosave.  Instead, on OK or Apply, save once.</string>
        </property>
        <property name="text">
         <string>Sa&amp;ve Once</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_autosave">
        <property name="toolTip">
         <string>Save image on timer</string>
        </property>
        <property name="whatsThis">
         <string>When checked, the image will be repeatedly saved using the indicated period.  This is useful, for example, for live web page images.</string>
        </property>
        <property name="text">
         <string>&amp;Autosave every</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_period">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Autosave timer</string>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_widthLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Width:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_xSize</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_xSize">
          <property name="whatsThis">
           <string>Width (in Pixels) of the image.</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_heightLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Height:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_ySize</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_ySize">
          <property name="whatsThis">
           <string>Height of the image (in pixels).</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>120</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>S&amp;pecify:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_comboBoxSizeOption</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_comboBoxSizeOption">
          <item>
           <property name="text">
            <string>width and height</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>size of square</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>width, maintaining aspect ratio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>height, maintaining aspect ratio</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>61</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="_autoExtension">
     <property name="text">
      <string>Automatic &amp;Extension</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::FileRequester</class>
   <extends>QWidget</extends>
   <header>filerequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_comboBoxFormats</tabstop>
  <tabstop>_comboBoxSizeOption</tabstop>
  <tabstop>_xSize</tabstop>
  <tabstop>_ySize</tabstop>
  <tabstop>_saveOnce</tabstop>
  <tabstop>_autosave</tabstop>
  <tabstop>_period</tabstop>
  <tabstop>_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_autosave</sender>
   <signal>toggled(bool)</signal>
   <receiver>_period</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
